DevOps/Kubernetes/create_eks_using_ekctl.md

```
@@ -0,0 +1,180 @@
+# Create EKS using EKCTL
+
+<aside>
+💡 install aws cli, ekctl,kubectl on client machine #machine requirment should be minimum 2 vcpu
+
+</aside>    
+
+```jsx
+# kubectl v 1.26
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)